toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -109,6 +111,11 @@ func (r *RpmRepoCloner) Initialize(destinationDir, tmpDir, workerTar, existingRp
 		logger.Log.Info("Enabling preview repo")
 	}
 
+	r.disableMarinerRepos = disableMarinerRepos
+	if disableMarinerRepos {
+		logger.Log.Info("Disabling Mariner repos")
```

Comment:
Thoughts about making this a warning, along with some details about what repos we are disabling?
Something like `logger.Log.Warnf("Disabling Mariner repos (repo ids:  %s", marinerReposGlob)`?

Reply:
That's fair, but this might require some extra work to get the full names of the repo ids. Either we have to parse the final repo file or parse the output of `tdnf repolist`, which isn't nicely formatted (?)

Reply:
Added this warning. The message on an example build is like this
```
WARN[0008] Disabling Mariner repos: [mariner-official-base mariner-official-base-debuginfo mariner-official-base-source]
```